src/collective/nitf/templates/media.pt

```diff
@@ -1,155 +1,154 @@
-            });
+                });
 
-            $('#sortable a.button.edit').prepOverlay({
+                $('#sortable a.button.edit').prepOverlay({
                     subtype: 'ajax',
                     filter: '#content',
                     formselector: 'form',
                     noform:'close'
-            });
-            $('#sortable a.button.delete').click(function(e){
-                e.preventDefault();
-                var $this = $(this);
-                $.ajax({
-                    url :$this.attr('href'),
-                    dataType:'json',
-                    success: function(data){
-                        if (data['status'] == 'success') {
-                          var p = $this.parents('li');
-                          p.fadeOut('slow', function(){
-                            p.remove();
-                          });
+                });
+                $('#sortable a.button.delete').click(function(e){
+                    e.preventDefault();
+                    var $this = $(this);
+                    $.ajax({
+                        url :$this.attr('href'),
+                        dataType:'json',
+                        success: function(data){
+                            if (data['status'] == 'success') {
+                                var p = $this.parents('li');
+                                p.fadeOut('slow', function(){
+                                    p.remove();
+                                });
+                            }
                         }
+                    });
+                });
+
+                $("#sortable").sortable({
+                    helper : 'clone',
+                    placeholder: "ui-state-highlight",
+                    update: function(event, ui){
+                        update_sortable(ui.item);
                     }
                 });
             });
-			
-            $("#sortable").sortable({
-                helper : 'clone',            
-                placeholder: "ui-state-highlight",
-                update: function(event, ui){
-                	update_sortable(ui.item);
-                }
-            });            
-	    });
-	    </script>
-    </metal:javascriptslot>
-</head>
+        </script>
+        </metal:javascriptslot>
+    </head>
 
-<body tal:define="errors options/state/getErrors">
-    <metal:main fill-slot="main">
+    <body tal:define="errors options/state/getErrors">
+        <metal:main fill-slot="main">
         <tal:protect tal:condition="python: not checkPermission('List folder contents', here)"
-                     tal:replace="here/raiseUnauthorized" />
+        tal:replace="here/raiseUnauthorized" />
 
         <metal:main_macro define-macro="main">
-            <metal:pic_sorter_macro define-macro="pic_sorter"
-                                    tal:define="media view/get_media;">
+        <metal:pic_sorter_macro define-macro="pic_sorter"
+        tal:define="media view/get_images;">
```

Comment:
This is actually the only change, but my editor remove all tabs and reindent code

Reply:
please commit only the change in that line.